Linux 386 builds missing syscall15XABI0 and syscall_syscall15X

### PureGo Version

57859678ab72

### Operating System

- [ ] Windows
- [ ] macOS
- [x] Linux
- [ ] FreeBSD
- [ ] NetBSD
- [ ] Android
- [ ] iOS

### Go Version (`go version`)

go version go1.25.4 darwin/arm64

### What steps will reproduce the problem?

I'm getting this build failure when I try to build for Linux 386 platforms

```
› GOOS=linux GOARCH=386 go build .
# github.com/ebitengine/purego
../../../.go/pkg/mod/github.com/ebitengine/purego@v0.10.0-alpha.3.0.20260115160133-57859678ab72/func.go:335:20: undefined: syscall15XABI0
../../../.go/pkg/mod/github.com/ebitengine/purego@v0.10.0-alpha.3.0.20260115160133-57859678ab72/func.go:346:20: undefined: syscall15XABI0
../../../.go/pkg/mod/github.com/ebitengine/purego@v0.10.0-alpha.3.0.20260115160133-57859678ab72/func.go:350:32: undefined: syscall_syscall15X
../../../.go/pkg/mod/github.com/ebitengine/purego@v0.10.0-alpha.3.0.20260115160133-57859678ab72/func.go:416:10: undefined: NewCallback
../../../.go/pkg/mod/github.com/ebitengine/purego@v0.10.0-alpha.3.0.20260115160133-57859678ab72/syscall.go:55:9: undefined: syscall_syscall15X
```

### What is the expected result?

PureGo to build on Linux/386

### What happens instead?

Missing syscalls

### Anything else you feel useful to add?

_No response_

`GOOS=linux GOARCH=386 go build .` seems to fail with older version like 0.8

```
% GOOS=linux GOARCH=386 go build .
# github.com/ebitengine/purego/internal/fakecgo
internal/fakecgo/callbacks.go:88:28: undefined: threadentry
internal/fakecgo/callbacks.go:89:28: undefined: x_cgo_init
internal/fakecgo/go_util.go:36:2: undefined: _cgo_sys_thread_start
```

~~Though linux/386 is in the tier 2, I think we should fix at least this compilation error.~~

--- a/README.md
+++ b/README.md
@@ -48,7 +48,7 @@ Tier 2 platforms are supported by PureGo on a best-effort basis. Critical bugs o
 #### Support Notes
 
 1. These architectures require CGO_ENABLED=1 to compile
-2. These architectures require special flags to compile with CGO_ENABLED=0
+2. These architectures require the special flag `-gcflags="github.com/ebitengine/purego/internal/fakecgo=-std"` to compile with CGO_ENABLED=0
 3. These architectures only support `SyscallN` and `NewCallback`
 4. These architectures are no longer supported as of Go 1.26
 

